CHANGELOG.asciidoc

```diff
@@ -14,6 +14,7 @@ https://github.com/elastic/beats/compare/v6.4.0...master[Check the HEAD diff]
 *Affecting all Beats*
 
 - Dissect syntax change, use * instead of ? when working with field reference. {issue}8054[8054]
+- Add `-e` flag by default to systemd unit so logs are collected by journald, behaviour can be reverted by overriding BEAT_LOG_OPTS environment variable with an empty value {pull}[]
```

Comment:
I think I'd word this as a more direct statement toward systemd users, like "On systems with systemd, the Beat logs are now written to journald by default rather than file. ..." Or some other wording that makes the breaking change get noticed.

Reply:
Thanks for the suggestion, changelog rephrased, added also to developers changelog.